internal/services/sentinel/sentinel_alert_rule_anomaly_data_source.go

```diff
@@ -0,0 +1,276 @@
+)
+
+type AlertRuleAnomalyDataSourceModel struct {
+	Name                         string                                  `tfschema:"name"`
+	DisplayName                  string                                  `tfschema:"display_name"`
+	WorkspaceId                  string                                  `tfschema:"log_analytics_workspace_id"`
+	AnomalyVersion               string                                  `tfschema:"anomaly_version"`
+	AnomalySettingsVersion       int32                                   `tfschema:"anomaly_settings_version"`
+	Description                  string                                  `tfschema:"description"`
+	Enabled                      bool                                    `tfschema:"enabled"`
+	Frequency                    string                                  `tfschema:"frequency"`
+	IsDefaultSettings            bool                                    `tfschema:"is_default_settings"`
+	RequiredDataConnectors       []AnomalyRuleRequiredDataConnectorModel `tfschema:"required_data_connector"`
+	SettingsDefinitionId         string                                  `tfschema:"settings_definition_id"`
+	Mode                         string                                  `tfschema:"mode"`
+	Tactics                      []string                                `tfschema:"tactics"`
+	Techniques                   []string                                `tfschema:"techniques"`
+	ThresholdObservation         []AnomalyRuleThresholdModel             `tfschema:"threshold_observation"`
+	MultiSelectObservation       []AnomalyRuleMultiSelectModel           `tfschema:"multi_select_observation"`
+	SingleSelectObservation      []AnomalyRuleSingleSelectModel          `tfschema:"single_select_observation"`
+	PrioritizeExcludeObservation []AnomalyRulePriorityModel              `tfschema:"prioritized_exclude_observation"`
+}
+
+type AlertRuleAnomalyDataSource struct{}
+
+var _ sdk.DataSource = AlertRuleAnomalyDataSource{}
+
+func (a AlertRuleAnomalyDataSource) Arguments() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+			ExactlyOneOf: []string{"name", "display_name"},
+		},
+
+		"display_name": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+			ExactlyOneOf: []string{"name", "display_name"},
+		},
+
+		"log_analytics_workspace_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ValidateFunc: workspaces.ValidateWorkspaceID,
+		},
+	}
+}
+
+func (a AlertRuleAnomalyDataSource) Attributes() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"anomaly_version": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"anomaly_settings_version": {
+			Type:     pluginsdk.TypeInt,
+			Computed: true,
+		},
+		"description": {
```

Comment:
could we put some lines here
```suggestion
		},

		"anomaly_settings_version": {
			Type:     pluginsdk.TypeInt,
			Computed: true,
		},

		"description": {
```